server/models/Request.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require("mongoose");
+
+const requestSchema = new mongoose.Schema(
+  {
+    user_id: {
+      type: String,
+      required: true,
+      unique: true,
+    },
+    sitter_id: {
+      type: String,
+      required: true,
+      unique: true,
+    },
```

Comment:
These fields are okay for now but when we implement a `User` model, ideally we change these to be references ([relevant mongoose doc](https://mongoosejs.com/docs/populate.html). Maybe we can leave a reminder about this using `TODO` comments here 🙌🏼

Reply:
Don't forget to camelCase our property keys 😄

Reply:
Updated with TODO